src/components/CommandBar/CommandBarHeader.tsx

```diff
@@ -130,7 +130,11 @@ function CommandBarHeader({ children }: React.PropsWithChildren<{}>) {
                           JSON.stringify(argValue)
                         )
                       ) : (
-                        <em>{argValue}</em>
+                        <em>
+                          {arg.valueSummary
+                            ? arg.valueSummary(argValue)
+                            : argValue}
+                        </em>
```

Comment:
Value summaries were only applied to `Object` argument payloads like `"selection"` type ones, but I wanted to provide truncation logic to the sample name's argument. This was a bug anyway, as you could set a `valueSummary` for an arg to a function that wasn't an `Object` type arg.